Source/Core/Core/DSP/DSPDisassembler.cpp

```diff
@@ -25,49 +25,15 @@ DSPDisassembler::DSPDisassembler(const AssemblerSettings& settings) : settings_(
 {
 }
 
-DSPDisassembler::~DSPDisassembler()
+bool DSPDisassembler::Disassemble(const std::vector<u16>& code, std::string& text)
 {
-  // Some old code for logging unknown ops.
-  std::string filename = File::GetUserPath(D_DUMPDSP_IDX) + "UnkOps.txt";
-  std::ofstream uo(filename);
-  if (!uo)
-    return;
-
-  int count = 0;
-  for (const auto& entry : unk_opcodes)
+  for (u16 pc = 0; pc < code.size();)
```

Comment:
Are you thinking that `DisassembleOpcode` might not increment `pc`, or that `code.size()` might be larger than what can be represented by a `u16`?